### PR TITLE
PD-402: Add Debug Kernel checkbox to SCALE docs

### PR DESCRIPTION
**Pull request by Mrt134**
- Add field to SCALE UI Reference Guide.
- Review the related tutorials and determine there is nothing special to add for this field.
- Update reference image of System Settings/Advanced to Cobia specific and remove 22.12 image.
- Add image of Kernel options card
- Optimize images.
- Move the generic System Settings/Advanced tutorial content to the _index file and rework slightly to function as a landing page.
- Alias from the removed URL to the new content location

Thanks for contributing to TrueNAS documentation! By opening a Pull Request, you're acknowledging that your changes will be distributed under the [Creative Commons 4.0](https://creativecommons.org/licenses/by-nc-sa/4.0/) license.

### CODE REVIEW

**Comment by Mrt134** [comment]
Waiting to merge this one until after @DjP-iX's #2018 is merged.

**Comment by Mrt134** [comment]
This PR is resynced and ready for review @DjP-iX. I left the _index.md file description more generic, but feel free to reset it to the more substantive sentence if you think that's more appropriate.

File: content/SCALE/SCALEUIReference/SystemSettings/AdvancedSettingsScreen.md [hunk 1]
```diff
@@ -77,6 +77,19 @@ The **Syslog** settings specify the logging level the system uses to record syst
...
+## Kernel Widget
+The **Kernel** widget shows options for configuring the Linux kernel installed with TrueNAS SCALE.
+
+![SCALE Kernel Options Card](/images/SCALE/23.10/SystemSettingsAdvancedKernel.png "Kernel Options Card") 
+
+{{< expand "Kernel Options (Click to expand)" "v" >}}
+{{< truetable >}}
+| Settings | Description |
+|----------|-------------|
+| Enable Debug Kernel | Set to boot a debug kernel after the next system reboot. |
```
Comment (micjohnson777) [inline content/SCALE/SCALEUIReference/SystemSettings/AdvancedSettingsScreen.md:89]:
```suggestion
| **Enable Debug Kernel** | Set to boot a debug kernel after the next system reboot. |
```

**Review by micjohnson777 — DISMISSED** [review]
Just one thing to fix.

**Review by DjP-iX — APPROVED** [review]

**Pull request merged**